README.md

```diff
@@ -0,0 +1,248 @@
+<div align="center">
```

Comment:
- [ ] I find myself unable to sign up/log in:

![image](https://github.com/Tracy-miranja/appoint-doctor-frontend/assets/57421823/ad9ffbda-cf3d-486d-b819-9002f558e0a0)

If this is a bug, kindly fix it.  If this is not a bug, this means your app needs special procedures to be run correctly.  If the latter is true, kindly add those instructions to your readme, so your users can log in into the app. 👍 


_note: This means I wasn't able to test the app's functionality.  This will be taken a look at the next review._

Reply:
I appreciate you for pointing this. it has been resolved